src/compiler/diagnosticMessages.json

```diff
@@ -2124,6 +2124,10 @@
         "category": "Error",
         "code": 2587
     },
+    "Cannot assign to '{0}' because it is a constant": {

```

Comment:
* Other error messages end in a period so this should too.
* We should also make the "read-only property" error message more specific, no need for "constant or"